scene/3d/skeleton_3d.cpp

```diff
@@ -847,12 +847,13 @@ void Skeleton3D::force_update_bone_children_transforms(int p_bone_idx) {
 	ERR_FAIL_INDEX(p_bone_idx, bone_size);
 
 	Bone *bonesptr = bones.ptrw();
-	List<int> bones_to_process = List<int>();
+	thread_local LocalVector<int> bones_to_process;
```

Comment:
I remember TLS giving us trouble on some compilers (especially mingw), but this might be finally solved now that we're using meganz's std threads implementation for mingw.

Still, CC @RandomShaper in case this rings a bell.

Reply:
This is fine. The same technique is already being used in other areas of rendering. Rationalizing out the use of `List` also matches an ongoing endeavor happening in other areas of the engine.

Reply:
Like Pedro said, we've had this technique in use in the renderers for a while so we should be clear.